f5_sphinx_theme/static/js/f5-theme.js

```diff
@@ -5,6 +5,8 @@ $(document).ready(function () {
   });
 });
 $(document).ready(function () {
-    $("#clouddocs-header").load("https://clouddocs.f5networks.net/header.html");
-    $("#clouddocs-footer").load("https://clouddocs.f5networks.net/footer.html");
+  var loc = window.location,
+      host = loc.protocol + '//' + loc.host;
+  $('#clouddocs-header').load(host + '/header.html');
```

Comment:
We are not loading header or footer locally. We are loading them from the clouddocs site so we no longer have duplicates and potential for the two to be out of sync. 

We can actually change the url to point to clouddocs.f5.com now, since we have HTTPS on it now.

Reply:
Ahh, I see what you're doing here now. That's fine. I had to pull them from staging before because we didn't have HTTPS on the production site yet.